src/Stryker.Core/Stryker.Core.UnitTest/Mutators/LinqMutatorTest.cs

```diff
@@ -67,8 +67,8 @@ static void Main(string[] args)
         [InlineData(LinqExpression.TakeWhile, LinqExpression.SkipWhile)]
         [InlineData(LinqExpression.Min, LinqExpression.Max)]
         [InlineData(LinqExpression.Max, LinqExpression.Min)]
-        [InlineData(LinqExpression.Sum, LinqExpression.Count)]
-        [InlineData(LinqExpression.Count, LinqExpression.Sum)]
+        [InlineData(LinqExpression.Sum, LinqExpression.Max)]
+        [InlineData(LinqExpression.Average, LinqExpression.Min)]
```

Comment:
I'm missing the other new mutations you added

Reply:
Done